app.js

```diff
@@ -127,9 +127,20 @@ if (Configuration.get('FEATURE_MORGAN_LOG_ENABLED')) {
 app.use(bodyParser.json({ limit: '10mb' }));
 app.use(bodyParser.urlencoded({ extended: true, limit: '10mb' }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, `build/${themeName}`)));
+
 app.use('/locales', express.static(path.join(__dirname, 'locales')));
 
+let themeAssetDir = path.join(__dirname, `build/${themeName}`);
```

Comment:
remove code duplication with https://github.com/hpi-schul-cloud/schulcloud-client/pull/2411/files#diff-86ec246c0db98dcda6ecd01fd13b114a98f6455fdd661e16324569e0064c0afbR12

Reply:
code optimization can be handled in a follow up, I will be busy with devops and org stuff till jan.